policy/release/cve.rego

```diff
@@ -181,12 +184,72 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
+# extracts the clair report attached to the image
+_clair_report := report if {
+	clair_task := lib.task_in_pipelinerun(_cve_scan_task_name)
+
+	tekton.is_trusted_task(clair_task)
```

Comment:
I don't think we need this. If there's an untrusted Task that behaves like the clair-scan Task, then there's no harm in evaluating its result as well. If a trusted version of the clair-scan Task must be included, then one can use the set of required tasks for that.